tests/fileTypeTests.py

```diff
@@ -119,7 +119,7 @@ def test_approp_mixed(self):
         """Test mixed job with some rows failing."""
         jobId = self.jobIdDict["mixed"]
         self.passed = self.run_test(
-            jobId, 200, "finished", 6330, 4, "complete", 46, True)
+            jobId, 200, "finished", 8847, 4, "complete", 46, True)
```

Comment:
Are these files larger now b/c the error message contains more info re: the specific fields and their values?